client/src/components/rapid/_utils/findRapidFromId.js

```diff
@@ -4,7 +4,7 @@
 
 const findRapidFromId = (linkId, allData) => {
   for (let rapid of allData.rapids) {
-    if (linkId == rapid.id) {
+    if (linkId === rapid.id) {
```

Comment:
Its important to make sure that you can use `===` equality. If `linkId` and `rapid.id` were not the same `type`, it would break. I think that this works, but always good to double check.

Reply:
I checked the type of `rapid.id`. It is a `String`. 
However, I did not find the type of `linkId`. It is passed from `arrow` array. 
`mockRivers.js` has it, but it's an empty array. Gotta check the DB for that.
Nevertheless, I assume it will be a `String` anyway, so I keep it like this for now.

Reply:
Ya your assumption is right.